### PR TITLE
tests: include new testing tools and utils

### PR DESCRIPTION
**Pull request by sergiocazzolato**
Using tools and utils that have been moved to snapd-testing-tools project.

The spread-shellcheck and check-test-format also have tests now.

### CODE REVIEW

**Review by Meulengracht — APPROVED** [review]
This looks good to me, LGTM.

File: .github/workflows/tests.yaml [hunk 1]
```diff
@@ -16,10 +16,15 @@ jobs:
 
     - name: Run shellCheck for tools
       run: |
+          # Run shellCheck 0.8
```
Comment (bboozzoo) [inline .github/workflows/tests.yaml]: the version listed here will become incorrect when a new shellcheck release is published. Maybe something in the lines of:
```
# run shellcheck from the snap which most likely is newer than what the distro provides
```

File: .github/workflows/tests.yaml [hunk 2]
```diff
@@ -16,10 +16,15 @@ jobs:
...
           sudo apt-get remove --purge shellcheck
           sudo snap install shellcheck
+          find tools utils -type f -exec sh -c "head -n 1 {} | egrep -a 'bin/bash|bin/sh' >/dev/null" \; -print -exec shellcheck {} \;
 
-          find tools -type f -exec sh -c "head -n 1 {} | egrep -a 'bin/bash|bin/sh' >/dev/null" \; -print -exec shellcheck {} \;
+          # Run shellCheck 0.7
```
Comment (bboozzoo) [inline .github/workflows/tests.yaml]:
```
# run shellcheck from the distro
```

File: .github/workflows/tests.yaml (hunk 1 above)
Comment (sergiocazzolato) [reply .github/workflows/tests.yaml]: nice, thanks

File: .github/workflows/tests.yaml (hunk 2 above)
Comment (sergiocazzolato) [reply .github/workflows/tests.yaml]: done

**Review by bboozzoo — APPROVED** [review]
LGTM, thanks!

**Pull request merged**